modin/pandas/test/utils.py

```diff
@@ -905,6 +911,12 @@ def execute_callable(fn, inplace=False, md_kwargs={}, pd_kwargs={}):
                         md_e, tuple(raising_exceptions)
                     ), f"not acceptable exception type: {md_e}"
                 elif raising_exceptions and type(raising_exceptions) is not type:
+                    if Engine.get() == "Ray":
+                        from ray.exceptions import RayTaskError
+
+                        # unwrap ray exceptions from remote worker
+                        if isinstance(md_e, RayTaskError):
```

Comment:
Do we have similar problems for Dask and Unidist?

Reply:
It seems not, otherwise they would have to break these tests too.